### PR TITLE
Stop hardcoding the name of the package in `pkg_resources.resource_st…

### PR DESCRIPTION
**Pull request by adamtheturtle**
…rings. This allows for vendoring `dcos_launch` under an alternative name

In DC/OS E2E we vendor DC/OS Launch.
`dcos_launch` in that environment is actually `dcos_e2e._vendor.dcos_launch`.
Right now we manually change the strings `'dcos_launch'` to `'dcos_e2e._vendor.dcos_launch'`.

This change should make no difference to `dcos_launch` and it should make the lives of us maintainers of DC/OS E2E a little easier. https://github.com/mesosphere/dcos-e2e/pull/695/files is the E2E PR which demonstrates the contributing instructions that can be removed given this change.

Usually I'd be wary of changing Launch for such a custom use case, but to me this is a simple change that does not add maintenance burden, I hope.

### CODE REVIEW

**Comment by falseperfection** [comment]
Does it not work to have `dcos_launch` be a library requirement that is pip-installable instead of vendored? I think the change is innocuous, except the intended practice of using `dcos-launch` as a library. @mellenburg probably has a more coherent thought on the subject.

**Comment by adamtheturtle** [comment]
@falseperfection We would like to ship DC/OS E2E as a `pip`-installable library (and Homebrew installable tool).

My understanding is that to do this and to have `dcos_launch` be a library requirement that is pinned and `pip`-installable requires our own users to use the deprecated `pip` option `--process-dependency-links`.

To avoid this we could ask that DC/OS Launch be released in a way that does not require us to do this. However, I considered that this very minor change is less burdensome than such a request.

**Comment by adamtheturtle** [comment]
> except the intended practice of using dcos-launch as a library

Are you saying that you worry about `dcos-launch` being used as a library at all? Isn't this what we discussed in a meeting with @mellenburg as a stop-gap solution?

**Comment by falseperfection** [comment]
> Are you saying that you worry about dcos-launch being used as a library at all?

The opposite. I worry about dcos-launch being vendored instead of used as a library. I'm 100% willing to acknowledge my concern may be unwarranted, but I have it so I said it before this trivial-seeming PR was merged. ¯\_(ツ)_/¯

**Comment by adamtheturtle** [comment]
@falseperfection Let's consider the options:

* Burden the users of DC/OS E2E (and users of any library which relies on DC/OS E2E) with a deprecated `pip` option. I'd rather not do this and I'd rather add a little more maintenance work for myself to achieve this. Note that we tried this before with DC/OS Test Utils and had to update Jenkins build configurations and TeamCity jobs, invoking multiple people.
* Come up with a way to not vendor and not use `--process-dependency-links` without changing dcos-launch - I'm all ears if you have ideas!
* Change DC/OS Launch in a way that means we can use pinned versions without vendoring or `--process-dependency-links`. That might mean doing releases, and is possibly quite a burden to the developers of dcos-launch.

**Comment by falseperfection** [comment]
Understood. IMO, if this is software we expect the world to use and not just internal, what's wrong with adding the burden of releases and managing it like every other software package out there?

**Comment by adamtheturtle** [comment]
@falseperfection That's what we do in DC/OS E2E - we even have a [CHANGELOG](https://github.com/mesosphere/dcos-e2e/blob/master/CHANGELOG.rst) and a completely automated release process. As a consumer of DC/OS Launch (and Test Utils) that is also my preference. However, I'd rather leave that choice to the maintainers of dcos-launch.

**Review by mellenburg — APPROVED** [review]
I think this is a pretty harmless change that will provide greater usability even if that use pattern isn't particularly recommended. We will definitely pursue the proper "burdensome" publishing steps for dcos-launch and dcos-test-utils as well but there is no reason for that to block this change

**Pull request merged**
